src/clusterfuzz/_internal/bot/tasks/utasks/minimize_task.py

```diff
@@ -600,10 +600,20 @@ def utask_main(uworker_input):
 
 def utask_postprocess(output):
   """Postprocess in a trusted bot."""
-  if output.error is not None:
+  if output.error:
     uworker_handle_errors.handle(output, HANDLED_ERRORS)
     return
 
+  # Since uworker_input is set automagically, this should never be true
+  if not output.uworker_input:
+    raise errors.BadStateError
+
+  # At this point, minimize_task_output should be set and
+  # its last_crash_result_dict field should not be empty
+  if not output.minimize_task_output \
+    or output.minimize_task_output.last_crash_result_dict == {}:
```

Comment:
Please change to `(if not output.minimize_task_output or not output.minimize_task_output.last_crash_result_dict)`
(we generally prefer paren over \

Reply:
I don't think it's so useful to check for something that will immediately throw an exception anyway, it's not really any harder to debug if this becomes an attribute error on 618, in fact I think it's easier, i would just test for attribute errors. Or maybe I'm wrong about this,

Reply:
ahhhhhh before when i wasnt serializing and deserializing, it didnt throw an attribution error so thats why i had added it but it should now. good - i didnt like these anyway lolol getting rid of it!